csharp/src/ProtocolBuffers.Test/TestProtos/FieldPresense.cs

```diff
@@ -0,0 +1,783 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Nit: file should be FieldPresence rather than FieldPresense. Or better, FieldPresenceTest given the name of the first class.